subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/HttpClientHelper.java

```diff
@@ -121,6 +121,9 @@ public boolean wasSuccessful(CloseableHttpResponse response) {
     }
 
     public CloseableHttpResponse performHttpRequest(HttpRequestBase request) throws IOException {
+        if(request.getURI().toString().contains("dl.bintray.com/kotlin")) {
```

Comment:
I guess we shouldn't commit this change to `master`.

Reply:
Yes, once this is approved, I'll remove this.